src/components/Block/Block.tsx

```diff
@@ -61,10 +64,9 @@ export interface BlockProps extends NestableBaseComponentProps {
   ellipsis?: boolean;
 }
 
-interface BlockStyles {
-  marginTop?: string;
-  paddingTop?: string;
-}
+const defaultContext = { textSize: undefined as TextSize | undefined };
```

Comment:
instead of using `as`, i'd create an interface for the context value

```
export interface BlockContext {
  textSize?: TextSize;
}

const defaultContext = {};
```